drone-webdriver/src/test/java/org/jboss/arquillian/drone/webdriver/binary/process/SeleniumServerTestCase.java

```diff
@@ -0,0 +1,104 @@
+package org.jboss.arquillian.drone.webdriver.binary.process;
+
+import org.assertj.core.api.Assertions;
+import org.jboss.arquillian.drone.webdriver.binary.handler.SeleniumServerBinaryHandler;
+import org.jboss.arquillian.test.spi.event.suite.AfterSuite;
+import org.jboss.arquillian.test.test.AbstractTestTestBase;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.openqa.selenium.remote.DesiredCapabilities;
+
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.OutputStream;
+import java.net.URL;
+import java.util.List;
+import java.util.logging.Handler;
+import java.util.logging.Logger;
+import java.util.logging.StreamHandler;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class SeleniumServerTestCase extends AbstractTestTestBase {
+
+    private static Logger log = Logger.getLogger(SeleniumServerExecutor.class.toString());
+    private OutputStream logCapturingStream;
+    private StreamHandler customLogHandler;
+
+    private String seleniumServerBinary;
+    private DesiredCapabilities capabilities;
+    private URL url;
+
+    @Override
+    protected void addExtensions(List<Class<?>> extensions) {
+        extensions.add(SeleniumServerExecutor.class);
+    }
+
+    @Before
+    public void initialise() throws Exception {
+        seleniumServerBinary =
+                new SeleniumServerBinaryHandler(new DesiredCapabilities()).downloadAndPrepare().toString();
+        capabilities = new DesiredCapabilities();
+        url = new URL("http://localhost:4444/wd/hub/");
+        attachLogCapture();
+    }
+
+    @Test
+    public void test_command_contains_serverArgs() throws Exception {
+
+        final String browser = "chrome";
+        final String seleniumServerArgs = "-debug";
+
+        fire(new StartSeleniumServer(seleniumServerBinary, browser, capabilities, url, seleniumServerArgs));
+
+        final String capturedLog = getTestCapturedLog();
+        final String output = parseLogger(capturedLog);
+
+        Assertions.assertThat(output.contains(seleniumServerArgs));
```

Comment:
I think that the correct format of the assertion should be: 
`Assertions.assertThat(output).contains(seleniumServerArgs);`
otherwise it doesn't verify anything. 
This applies also for all following occurrences in next tests...